plug.vim

```diff
@@ -139,6 +139,11 @@ function! s:define_commands()
   if !executable('git')
     return s:err('`git` executable not found. Most commands will not be available. To suppress this message, prepend `silent!` to `call plug#begin(...)`.')
   endif
+  if !has('nvim')
+    \ && (has('win32') || has('win32unix'))
+    \ && (!has('multi-byte') || &encoding !=# 'utf-8')
+    return s:err('Only Vim with +multi_byte feature is supported. Add `set encoding=utf-8` in your vimrc before running `plug#begin()`.')
```

Comment:
@mattn Is this a reasonable restriction to guarantee that the batchfile has `utf-8` encoding?

Reply:
Why do you check support of encoding=utf-8 ?

Reply:
I want to guarantee that `writefile` works with unicode and batchfile has unicode encoding. I don't know enough about the Vim internals to know if this is needed or is insufficent so I'm asking. Maybe `writefile` with `b` flag is enough?

Reply:
The batch file must be written in code page NOT unicode. So you can use iconv().

Reply:
```
let cp = libcallnr('kernel32.dll', 'GetACP', 0)
let text = iconv(text, &encoding, printf('cp%d', cp))
```

Reply:
iconv  replaces unicode characters with `?`. I'd rather use powershell if the batchfile cannot be executed with unicode characters and non-unicode codepage.

Reply:
Why you have to handle unicode characters?